src/blockdata/transaction.rs

```diff
@@ -608,7 +608,23 @@ impl Decodable for Transaction {
     }
 }
 
-/// Hashtype of a transaction, encoded in the last byte of a signature
+/// Error occuring when parsing a SigHashType
+#[derive(Debug, PartialEq)]
+pub enum ParseSigHashTypeError {
```

Comment:
According to the general consensus in #555 there should be a few more derives (whatever makes sense in this case, `Clone` and `Copy` certainly do and a few more probably too).